### DISCUSSION TITLE
Installing through epic games launcher gets stuck at "cleaning up...."

### DISCUSSION THREAD

**Discussion by srlegrand** [TheHerobrine/DarkerNodes#99] [Q&A]
And now my Unreal editor stops loading at 39%. 
![image](https://user-images.githubusercontent.com/44444226/104814362-5d1a8800-5806-11eb-920f-e8023c6a2a9c.png)

**Comment by hugoattal** [comment]
That's weird 🤔... I believe it has to do with the launcher more than the plugin though.

Did you try to force close the Epic Games Launcher and restart it?

**Reply by srlegrand** [reply]
I did. I also had to kill both UE4 and launcher processes as I could not close them anymore. I restarted and the same problem happens. I think I may have to reinstall the engine for me to be able to launch it again. :/ I'll let you know if I find out more.

**Reply by hugoattal** [reply]
@srlegrand Maybe try deleting the plugin 🤔 ?
It's in `D:\Program Files\Epic Games\UE_4.26\Engine\Plugins\Marketplace\DarkerNodes`

**Reply by srlegrand** [reply]
Thanks for the info. It seems that after a couple of restarts it worked as expected. Just loaded the plugin successfully in my projects. Thank you! I really like this new look. I absolutely hate the default gradients and drop-shadows. :) Good work!

**Reply by hugoattal** [reply]
Thank you very much 😄 !
